timesync

```diff
@@ -99,7 +103,27 @@ def parse_all_args():
     return parser.parse_args()
 
 
+def read_config():
+    """Returns a dictionary representing the contents of the json file
+    $HOME/timesyncconf.json or a set of sensible defaults if that file does
+    not exist.
+    """
+    sensible_defaults = {
+        'url': 'http://localhost:3000',
+        'editor': 'vi',
```

Comment:
Yup, totally. I think the mentioned feature should probably include that 'editor' k/v when the feature is added then.